### ISSUE TITLE
Remove reference to pilot-hub image in our docs

### ISSUE THREAD

**Issue by yuvipanda** [2i2c-org/infrastructure#1815]
@yuvipanda Did this image used to be called `pilot-hub`? I am thinking we may need to

* clean the old repo out of quay.io https://quay.io/organization/2i2c (captured in https://github.com/2i2c-org/infrastructure/issues/1840)

- [x] update the docs here as that still references `pilot-hub`: https://infrastructure.2i2c.org/en/latest/topic/hub-image.html

_Originally posted by @sgibson91 in https://github.com/2i2c-org/infrastructure/pull/1791#discussion_r1004496202_

**Comment by sgibson91** [comment]
Docs fixed in [`d365c77` (#1814)](https://github.com/2i2c-org/infrastructure/pull/1814/commits/d365c7771da80fe7577176f084b6428bad89d6a7)

**Comment by sgibson91** [comment]
I believe this to be the image repo to delete on quay.io as well https://quay.io/repository/2i2c/pilot-hub

**Comment by damianavila** [comment]
> I would recommend a new issue tracking renaming that image (and the quay.io repo) to something more descriptive/less confusing.

Done: https://github.com/2i2c-org/infrastructure/issues/1840.

**Comment by damianavila** [comment]
Since we decoupled the two items at the top, should we close this one once the PR with the docs lands? Or there is anything else to track here, @sgibson91?

**Comment by sgibson91** [comment]
I think the only reason this issue got opened in the first place is because I was confused by the naming of stuff and what docs referred to which images. I think we can close this now.
